bin/bossplot

```diff
@@ -72,6 +97,14 @@ def main():
         help = 'Show the subtracted sky flux instead of the object flux.')
     parser.add_argument('--add-sky', action = 'store_true',
         help = 'Add the subtracted sky to the object flux (overrides show-sky).')
+    parser.add_argument('--flux-range', type=str, default='0.5%:99.5%', metavar="MIN:MAX",
```

Comment:
There seem to be fewer deep troughs that big spikes, but the extremes of wavelength usually have noise that goes much lower than the rest of the minimums; probably worth chopping that off, keeping the 0.5% minimum.